test-network/network.sh

```diff
@@ -422,6 +437,14 @@ while [[ $# -ge 1 ]] ; do
     CC_INIT_FCN="$2"
     shift
     ;;
+  -cport )
+    CC_SERVER_PORT="$2"
+    shift
+    ;;
+  -ccaasdr )
```

Comment:
Maybe simply `ccaas`? I at least recognize that term. `ccaasdr` doesn't mean anything to me.

Reply:
ccaas though doesn't really reflect what this option means. 

--chaincode-as-a-service-do-not-start-docker

would be the full option name :-)